server/service/integration_mdm_lifecycle_test.go

```diff
@@ -710,13 +719,24 @@ func (s *integrationMDMTestSuite) TestLifecycleSCEPCertExpiration() {
 	require.NoError(t, err)
 	require.Nil(t, cmd)
 
+	fmt.Println(">>> manual host: ", manualHost.UUID)
+	fmt.Println(">>> automatic host: ", automaticHost.UUID)
+	fmt.Println(">>> automatic with ref host: ", automaticHostWithRef.UUID)
+	fmt.Println(">>> migrated device host: ", migratedDevice.UUID)
+
 	expireCerts := func() {
 		mysql.ExecAdhocSQL(t, s.ds, func(q sqlx.ExtContext) error {
-			_, err := q.ExecContext(ctx, `
+			res, err := q.ExecContext(ctx, `
                   UPDATE nano_cert_auth_associations
-                  SET cert_not_valid_after = DATE_SUB(CURDATE(), INTERVAL 1 YEAR)
+                  SET cert_not_valid_after = DATE_SUB(CURDATE(), INTERVAL 400 DAY)
```

Comment:
I changed that when testing different things, I don't think that matters - 1 YEAR should work too.